packages/front-generator/src/generators/react-typescript/common/entity.ts

```diff
@@ -0,0 +1,31 @@
+import {Entity, EntityAttribute, ProjectModel, ViewProperty} from "../../../common/model/cuba-model";
+import {collectAttributesFromHierarchy} from "../../../common/model/cuba-model-utils";
+import {EntityTemplateModel, getEntityPath} from "./template-model";
+
+export function createEntityTemplateModel(entity: Entity, projectModel: ProjectModel) {
+  return {
+    ...entity,
+    path: getEntityPath(entity, projectModel)
+  };
+}
+
+export function determineDisplayedAttributes(
```

Comment:
Why not name it 'getDisplayedAttributes'?

Reply:
done